python/lsst/sims/maf/driver/runBundle.py

```diff
@@ -0,0 +1,96 @@
+import os, re, copy, warnings
+import lsst.sims.maf.sliceMetrics as sliceMetrics
+import lsst.sims.maf.db as db
+import lsst.sims.maf.utils as utils
+
+def runBundle(mafBundle, verbose=True, makePlots=True):
+    """
+    mafBundle should be a dict with the following key:value pairs:
+    'metricList':List of MAF metric objects
+    'slicer': instatiated MAF slicer
+    'dbAddress': string that gives the address to the desired DB
+    'sqlWhere': string that gives the sql where clause for pulling the data
+    optional keys:
+    'outDir': Output directory
+    'stackerList': List of configured MAF stackers
```

Comment:
nitpick: move this up int he list of arguments: 
metric, slicer, stackerList, sqlWhere, opsimdb, runName, metadata, outDir
seems a more logical order (and leads people down the list of critical (and evaluating now) -> supporting (necessary, but wider usage) ->  decorative/helpful metadata.
(under which, outDir could go earlier after opsimdb .. but since it's very different than the rest, I like it last).